packages/common/src/controllers/AccountController.ts

```diff
@@ -258,86 +253,68 @@ export default class AccountController {
 
   // TODO: Decide if it's worth keeping this or just leave combined with getUser
   // noinspection JSUnusedGlobalSymbols
-  getCustomerConsents = async (): Promise<GetCustomerConsentsResponse> => {
+  getCustomerConsents = async () => {
     const { getAccountInfo } = useAccountStore.getState();
-    const { config } = useConfigStore.getState();
     const { customer } = getAccountInfo();
 
-    const response = await this.accountService.getCustomerConsents({ config, customer });
+    const consents = await this.accountService.getCustomerConsents({ customer });
 
-    if (response?.consents) {
-      useAccountStore.setState({ customerConsents: response.consents });
+    if (consents) {
+      useAccountStore.setState({ customerConsents: consents });
     }
 
-    return response;
+    return consents;
   };
 
-  getPublisherConsents = async (): Promise<GetPublisherConsentsResponse> => {
+  getPublisherConsents = async () => {
     const { config } = useConfigStore.getState();
 
-    const response = await this.accountService.getPublisherConsents(config);
-
-    useAccountStore.setState({ publisherConsents: response.consents });
+    useAccountStore.setState({ loading: true });
+    const consents = await this.accountService.getPublisherConsents(config);
 
-    return response;
+    useAccountStore.setState({ publisherConsents: consents, loading: false });
```

Comment:
Why the loading false here? Seems out of place...

Reply:
Yes, we've made the AccountStore the source of truth for consents, because the consents were stored in the AccountStore but also used as useQuery result.

But I agree that the loading state is now "misused" even more. We should add separate `loading` states for these cases in the AccountStore.

We'll look into this.